lisp/ess-r-mode.el

```diff
@@ -51,6 +51,7 @@
 (autoload 'ess-r-args-show      "ess-r-args" "(Autoload)" t)
 (autoload 'ess-r-args-auto-show "ess-r-args" "(Autoload)" t)
 
+(declare-function ess-sqpe-versions-create "ess-sp6w-d")
```

Comment:
Could you move this inside `ess-r-s-versions-creation` please.

Reply:
Done